sdk/core/src/_helpers.ts

```diff
@@ -17,3 +20,9 @@ export const stripParamsFromDidUrl = (did: string): string =>
 export function isW3cCredential(credential: any): boolean {
   return !!credential.type
 }
+
+interface SDKHeaders extends IncomingHttpHeaders {
+  'X-SDK-Version': string
+}
+
+export const createApiServiceHeaders = (): SDKHeaders => ({ 'X-SDK-Version': version })
```

Comment:
Why do we need to extend `IncomingHttpHeaders` then? SDKHeaders could just contain a single field.

Reply:
That made in case we would like to set some predefined headers in future, but can be removed for now and added when we need it.